src/components/QuizGameQuestions/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./index.css";
+
+const QuestionCard = ({
+  question,
+  answers,
```

Comment:
Does answer mean the stored correct answer to the quiz? I would say so but it took me few seconds to understand what's happening and I'm still not sure why it ends with an "s". maybe correctAnswer would be more readable

Reply:
as we have 10  questions each round i used answers, and didnot use correct answers because we have correct and not correct answers

Reply:
so is it the users answers?


in general every feedback about readability should either result in a better variable name or a comment

Reply:
I think I get what this is saying now but you need to think of a more appropriate variable name, answers is very vague

Reply:
PTAL